parlai/agents/seq2seq/modules.py

```diff
@@ -117,7 +118,8 @@ def forward(self, xs, ys=None, cands=None, valid_cands=None):
 class Encoder(nn.Module):
     def __init__(self, num_features, padding_idx=0, rnn_class='lstm',
                  emb_size=128, hidden_size=128, num_layers=2, dropout=0.1,
-                 bidirectional=False, shared_lt=None, shared_rnn=None):
+                 bidirectional=False, shared_lt=None, shared_rnn=None,
+                 sparse=False):
```

Comment:
Why do we have this sparse argument here (and in the decoder) when it is always initialized to False?

Reply:
I'd like for it to be an option, but there are some features still missing--this was the first step towards supporting it. maybe it's too soon to commit this section...